tremor-script/src/ast/raw.rs

```diff
@@ -1256,9 +1256,17 @@ where
     fn up<'registry>(self, helper: &mut Helper<'script, 'registry>) -> Result<Self::Target> {
         // We compute the target before shadowing the key and value
 
+        if self.fold.is_some() {
+            helper.warn(
+                self.extent().expand_lines(2),
+                self.extent(),
+                &"The `use <op>` syntax is experimental and may be deprecated or replaced",
```

Comment:
👍  This is great, keeps the door open for [H|T] functions and functional fold down the road